examples/basic-with-mini-css-plugin/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
```

Comment:
Shouldn't it we use `env` instead of `es2015`. `es2015` is deprecated

Reply:
yes you should use env